src/EventSubscriber/TideCoreRouteAlter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\tide_core\EventSubscriber;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Class TideCoreRouteAlter.
+ *
+ * @package Drupal\tide_core
+ */
+class TideCoreRouteAlter extends RouteSubscriberBase {
+
+  /**
+   * Alter scheduled_transitions route and path.
+   *
+   * {@inheritDoc}.
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+    $route = $collection->get('entity.node.scheduled_transitions');
+    if ($route) {
+      $route->setPath('/node/{node}/scheduled-publishing');
```

Comment:
@vincent-gao If we change the path here, any other modules relying on the default path may get issues with broken links.

Reply:
fixed.

Reply:
thanks @sonnykt